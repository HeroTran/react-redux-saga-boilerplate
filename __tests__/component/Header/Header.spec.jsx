
/*
import React from 'react';
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';
import Header from '../../../app/renderer/components/Common/Header/Header';

describe('should be rendered without error', () => {
  it('matches snapshot', () => {
    const tree = renderer.create(<MemoryRouter><Header /></MemoryRouter>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
*/