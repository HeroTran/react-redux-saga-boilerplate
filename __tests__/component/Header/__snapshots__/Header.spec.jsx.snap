// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should be rendered without error matches snapshot 1`] = `null`;
